source/utilities.tex

```diff
@@ -5896,7 +5892,7 @@
 \begin{itemdescr}
 \pnum
 \effects
-Constructs an object of class \tcode{bitset<N>} as if by:
+As if by:
```

Comment:
LWG has been replacing "as if by:" with "Equivalent to:" in newer wording, but for cases like this (constructor is equivalent to other constructor) we simply use constructor delegation in the class synopsis and avoid prose specification.

Reply:
-> separate editorial issue, please.

Reply:
We have historically considered changing "As if by" to "Equivalent to" to not be editorial, because "Equivalent to" has implications for the other descriptive elements as described in [structure.specifications]p4 which are not (clearly) implied by "As if by" wording.